src/style.css

```diff
@@ -0,0 +1,120 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  --card__link-color: #00acdc;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 30px 14px 15px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 45px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__name-product {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-bottom: 6px;
+}
```

Comment:
You can separate the properties of the CSS to make them easier to read)
For example, first the position, then the block system, and then the printing house)